briefcase/app.py

```diff
@@ -8,12 +8,16 @@
 import subprocess
 import shutil
 import sys
+import textwrap
 import uuid
 
 from datetime import date
 from distutils.core import Command
 
 import pip
+import pkg_resources
+from setuptools.command import easy_install
```

Comment:
/me has a Pavlovian shudder at the utterance of these two magic words... :-)

Reply:
I know, but this is the version that setuptools itself ships to use, so I felt ok about the functional reuse rather than reimplement :-)

Reply:
True. But old wounds... :-)